### ISSUE TITLE
W3C compliance

### ISSUE THREAD

**Issue by extendednotes** [orestbida/cookieconsent#311]
The documentation says we have to set the script type to _"text/plain"_. Unfortunately this is not compliant with W3C specifications in combination with src. I get the following error:

**_Error: A script element with a src attribute must not have a type attribute whose value is anything other than the empty string, a JavaScript MIME type, or module._**

[More information here](https://rocketvalidator.com/html-validation/a-script-element-with-a-src-attribute-must-not-have-a-type-attribute-whose-value-is-anything-other-than-the-empty-string-a-javascript-mime-type-or-module)

My script definition:
`<script type="text/plain" data-cookiecategory="analytics" src="https://js.stripe.com/v3/" data-turbo-track="reload" defer></script>`

**Comment by orestbida** [comment]
Duplicate of #189.

TLDR; you can change `src` to `data-src`.
